Changelog.md

```diff
@@ -6,6 +6,7 @@ Language Features:
 
 Compiler Features:
  * Commandline Interface: Use proper severity and coloring also for error messages produced outside of the compilation pipeline.
+ * EVM: Deprecate support for "Constantinople" and older EVM versions.
```

Comment:
Maybe we should deprecate `petersburg` as well? https://github.com/ethereum/solidity/issues/14475#issuecomment-1750986407

Reply:
In my mind, the plan was actually to deprecate versions **older** than constantinople, **not including** constantinople itself... not sure why I made it ``<=`` in the title of https://github.com/ethereum/solidity/issues/14475

Reply:
Changed.